packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapController.m

```diff
@@ -726,6 +728,14 @@ - (nullable FlutterStandardTypedData *)takeSnapshotWithError:
   return imageData ? [FlutterStandardTypedData typedDataWithBytes:imageData] : nil;
 }
 
+- (nullable NSNumber *)isAdvancedMarkersAvailable:
+    (FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  NSUInteger advancedMarkerFlag =
+      self.controller.mapView.mapCapabilities & GMSMapCapabilityFlagsAdvancedMarkers;
+  BOOL boolValue = advancedMarkerFlag != 0;
+  return [NSNumber numberWithBool:boolValue];
```

Comment:
Optional: `return [NSNumber numberWithBool:(advancedMarkerFlag != 0)];`. Or alternately, `return @(boolValue);`

(Don't do both though; Obj-C does weird things with box syntax and inline expressions that should evaluate to bools, which can cause type problems when run through the method channels.)